yeat/workflow/Pacbio

```diff
@@ -98,8 +98,6 @@ rule metamdbg:
         contigs="analysis/{sample}/{readtype,pacbio-raw|pacbio-corr|pacbio-hifi}/{label}/metamdbg/contigs.fasta"
     input:
         reads="seq/input/{sample}/{readtype}/combined-reads.fq.gz"
-    conda:
-        "yeat-metaMDBG"
```

Comment:
`metaMDBG` can now be installed with conda. No need to create a new dedicated conda environment.